auth/src/api/handlers.rs

```diff
@@ -0,0 +1,45 @@
+use axum::{
+    extract::{Path, State},
+    Json,
+};
+use tracing::instrument;
+
+use crate::{
+    api::builder::RouterState,
+    error::Error,
+    user::{AccountName, Admin, User, UserManagement},
+};
+
+#[instrument(skip(user_manager))]
+pub(crate) async fn get_user(
+    _: Admin,
+    State(RouterState { user_manager }): State<RouterState>,
+    Path(account_name): Path<AccountName>,
+) -> Result<Json<User>, Error> {
+    let user = user_manager.get_user(account_name).await?;
+    // TODO: use the user response struct in common, or create a new one
```

Comment:
Not quite sure what the desired response is here, should we leave it as User for now?

Reply:
We might want to interact with this in the future from `shuttle-admin`, so let's pull the response from common